.github/workflows/auto-build.yml

```diff
@@ -23,9 +27,7 @@ jobs:
         uses: ./.github/actions/build_setup
 
       - name: Build
-        uses: gradle/gradle-build-action@v2
-        with:
-          arguments: build
+        run: ./gradlew build --build-cache
```

Comment:
The Gradle build action will run this with the build cache automatically, and will utilize its own caching. Not sure why this was changed.

Reply:
gradle build action is deprecated.